src/routes/components/components.json

```
@@ -2347,5 +2347,13 @@
 		"category": "Stores",
 		"tags": ["in-page navigation","stores and state"],
 		"stars": 1
+	}, {
+	   	"title": "@joeinnes/svelte-image",
```

you probably need to do something like this though it'd be safer to just run `npm run format`
```suggestion
	},
	{
	   	"title": "@joeinnes/svelte-image",
```